client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -85,11 +89,25 @@ const DesignSelector: React.FunctionComponent = () => {
 								} }
 							>
 								<span className="design-selector__image-frame">
-									<img
-										alt=""
-										aria-labelledby={ makeOptionId( design ) }
-										src={ getDesignImageUrl( design ) }
-									/>
+									{
```

Comment:
These curly braces should probably go.

Reply:
I think I'll just remove the comment completely, was moved from somewhere else but its probably no longer valid.